### PR TITLE
[oneTBB] task_group contructor acepting task_group_context

### PR DESCRIPTION
**Pull request by anton-potapov**
- added `task_group` constructor accepting `task_group_context`

### CODE REVIEW

File: source/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .. SPDX-FileCopyrightText: 2019-2020 Intel Corporation
```
Comment (alexandraepan) [inline source/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.rst:1]:
```suggestion
.. SPDX-FileCopyrightText: 2019-2021 Intel Corporation
```

File: source/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-..
+......................................................
```
Comment (alexandraepan) [inline source/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.rst:2]:
```suggestion
..
```

Comment (alexandraepan) [reply source/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.rst:2]: To fix CI issue

**Comment by anton-potapov** [comment]
will be a part of cumulative PR with task_group_extensions

**Pull request closed without merging**